### ISSUE TITLE
###### PLEASE ADD BACK THE PLAYER SKIN BAR TO THE VILLAGER EDITOR ######

### ISSUE THREAD

**Issue by joelGemini976** [WildBamaBoy/minecraft-comes-alive#1289]
I am sorry if this is in your way i have been playing your mod for years creating my 100 baby world. I never finished the world, so when i went back to finish or play more on that world because i used the player skins bar on all of my "children" and villagers, they were all Steve skins. I was so devastated all of my work of 79 children was ruined ( this is no shade just that i was hella sad ) Also i loved playing your mod to use the player skin bar as an extra boost of fun to see all my friends skins in my game and i just loved it. Also i know you can add skins to the textures but i have tried it many times and i cant find any skin that works and i like. I don't know why you took it out i cant find a source that explains it to me it was never in the way and i hope you see this, it would first of all be an honor and second of all i would really appreciate if you could add it back it would be awesome. I hope you get this.

**Comment by Ginger60** [comment]
I'm having the same problem too.  I had to edit my skins to the "converted" style.  I had to download a program called "SkinConvertingSheep". Just drag and drop your skins and hit Convert.  I renamed my edited skins with the "_converted" at the end. But now my problem is that the game doesn't "see" them.

**Comment by keizaaldragon** [comment]
I agree, I had a lot of fun playing MCA in multiplayer with the villager editor. It was fun cranking up their height and width scale x100 and giving them a Cthulhu skin. I'm curious as to why this feature was removed, it was really great fun.

**Comment by teagan75** [comment]
I cannot even change the trades using the editor or the skins included with the mod so I think the editor is just not working atm

Edit: I did get the skins and careers to finally change BUT the trades do not change
